Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/2024-rest-foods-backend-2.iml\" filepath=\"$PROJECT_DIR$/.idea/2024-rest-foods-backend-2.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/rest-foods-backend-2-2024.main.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/rest-foods-backend-2-2024.main.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision cb984aa0ccd6fbf0b4ca69e9d0a86d8521704b10)
+++ b/.idea/modules.xml	(date 1749723017826)
@@ -3,7 +3,6 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/2024-rest-foods-backend-2.iml" filepath="$PROJECT_DIR$/.idea/2024-rest-foods-backend-2.iml" />
-      <module fileurl="file://$PROJECT_DIR$/.idea/modules/rest-foods-backend-2-2024.main.iml" filepath="$PROJECT_DIR$/.idea/modules/rest-foods-backend-2-2024.main.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
